docs/hub/rl-baselines3-zoo.md

```diff
@@ -0,0 +1,55 @@
+# Using RL-Baselines3-Zoo at Hugging Face
+
+`rl-baselines3-zoo` is a training framework for Reinforcement Learning, using Stable Baselines3.
+
+## Exploring RL-Baselines3-Zoo in the Hub
+
+You can find RL-Baselines3-Zoo models by filtering at the left of the [models page](https://huggingface.co/models?library=stable-baselines3).
+
+The Stable-Baselines3 team is hosting a collection of +150 trained Reinforcement Learning agents with tuned hyperparameters that you can find [here](https://huggingface.co/sb3).
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
+2. Metadata tags that help for discoverability.
+3. A video widget where you can watch your agent performing.
+
+## Using existing models
+You can simply download a model from the Hub using `load_from_hub`:
+
+If it's from the official Stable-Baselines3 team:
+
+```
+# Download ppo Cartpole model from the Stable-Baselines3 team
+python -m utils.load_from_hub --algo ppo --env CartPole-v1 -f logs/ -orga sb3
+```
+
+Else:
+
+```
+# Download model and save it into the logs/ folder
+python -m utils.load_from_hub --algo dqn --env SpaceInvadersNoFrameskip-v4 -orga ThomasSimonini -f logs/
+python enjoy.py --algo dqn --env SpaceInvadersNoFrameskip-v4  -f logs/
+```
```

Comment:
I think you can merge the two of them as the code is the same, you just change the username/org

Reply:
✔️